Feature/forms (#241)
infinum/eightshift-docs: website/forms/intro.md

---
id: intro
title: Intro
---

> Eightshift forms plugin is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.

Reviewer: ```suggestion
Eightshift Forms is a full-featured form builder plugin that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```

> Eightshift forms plugins is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.

Reviewer 2: ```suggestion
Eightshift forms plugin is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```

> Eightshift forms plugins is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.

Reviewer 3: ```suggestion
Eightshift forms plugin is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```
